packages/cosmic-proto/src/address-hooks.js

```diff
@@ -189,9 +197,10 @@ export const decodeAddressHook = (addressHook, charLimit) => {
   }
 
   /** @type {HookQuery} */
-  const query = queryString.parse(hookStr);
-  return { baseAddress, query };
+  const query = { ...queryString.parse(hookStr) }; // spread to remove null prototype
```

Comment:
`mustMatch` thought this was a remotable without this change. A bit over my head but this seems to work.

Reply:
>  A bit over my head

I've come to learn `Object.create(null);` or `Object.setPrototypeOf({}, null)` is a performance optimization. It provides an object with no methods that's useful for dictionaries / hashmaps